lib/Cake/Model/Datasource/Database/Sqlite.php

```diff
@@ -591,4 +591,14 @@ public function nestedTransactionSupported() {
 		return $this->useNestedTransactions && version_compare($this->getVersion(), '3.6.8', '>=');
 	}
 
+/**
+ * Returns a locking hint for the given mode.
+ * Sqlite Datasource doesn't support row-level locking.
```

Comment:
Separate with a blank line. So the first becomes the short description, the second the "long" description.